TopMovies/API/GraphQL/NetworkManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  NetworkManager.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 01.04.2021.
+//
+
+import Foundation
+
+struct QueryJSON: Encodable {
+  let query: String
+}
+
+class NetworkManager {
```

Comment:
Why manager? It doesn't look like it manages some other entities inside. Any other suggestions?)